doc/source/user/plugins.rst

```diff
@@ -0,0 +1,233 @@
+Windows. You can also explicitly specify the
+:attr:`~watts.PluginSAS.executable`::
+
+    sas_plugin.executable = "/path/to/sas-exec"
+
+Furthermore, the paths to the SAS utilities that convert the ".dat" files to
+".csv" files must be specified with the :attr:`~watts.PluginSAS.conv_channel`
+and :attr:`~watts.PluginSAS.conv_primar4` attributes::
+
+    sas_plugin.conv_channel  = "/path/to/CHANNELtoCSV.x"
+    sas_plugin.conv_primar4  = "/path/to/PRIMAR4toCSV.x"
+
+Similar to the SAS executable, the utilities are also OS-dependent. To execute
+SAS, the :meth:`~watts.PluginSAS` instance is called directly in the same way as
+other plugins.
+
+RELAP5-3D Plugin
+++++++++++++++++
+
+The :class:`~watts.PluginRELAP5` class handles execution of RELAP5-3D. Note that
+the plugin is designed for the execution of RELAP5-3D v4.3.4 and thus may not be
+compatible with other version of RELAP5-3D. RELAP5-3D uses text-based input
+files that can be templated as follows:
+
+.. code-block:: jinja
+
+    *                 Time         Power
+    20250001          -1.0         0.0
+    20250002           0.0      {{ heater_power_1 }}
+    20250003         1.0e3      {{ heater_power_2 }}
+
+If the templated input file is `relap5_template`, then the RELAP5-3D plugin can be
+instantiated with the following command line::
+
+    relap5_plugin = watts.PluginRELAP5('relap5_template')
+
+RELAP5-3D requires the executable, license key, and the input file to be in the
+same directory to run. Thus, before running the RELAP5-3D plugin, you need to
+specify the directory that the executable and the license key are in (they must
+be in the same directory). This can be done by adding the ``RELAP5_DIR``
+variable to the environment or by explicitly specifying the path in the Python
+script as::
+
+    relap5_plugin.relap5_dir = "/path/to/relap5_dir/"
+
+The RELAP5 executable is OS-dependent. It defaults to ``relap5.x`` (assumed to
+be present on your :envvar:`PATH`) for Linux and macOS, and ``relap5.exe`` for
+Windows.
+
+As with other plugins, extra input files and templates can be specified as
+described in :ref:`input_files`. Note that the fluid property files can be
+specified via ``extra_args``. Another approach is to simply put them in the same
+directory as the executable and license key before running the plugin.
+
+For the postprocessing of RELAP5-3D results, the plugin converts the default
+"plotfl" plot file generated by RELAP5-3D into a ".CSV" file. Card-104 must be
+specified as "ascii" in the RELAP5-3D input file as::
+
+    104          ascii
+
+to ensure that the "plotfl" is in ASCII format instead of the default binary
+format. As the conversion process could be computationally expensive, user can
+turn it off by omitting Card-104 in the RELAP5-3D input file and adding
+``plotfl_to_csv=False`` when instantiating the plugin as follows::
+
+    relap5_plugin = watts.PluginRELAP5('relap5_template', plotfl_to_csv=False)
+
+MCNP Plugin
```

Comment:
Thanks for adding initial documentation. I assume that in next PR, we will addd more information about MCNP and Serpent plugin - I understand the current PR only provides minimal plugin.

Reply:
The purpose of these sections (in plugins.rst) is to describe the _unique_ features of each plugin, as opposed to the general, shared features that are already described in usage.rst. Right now, because the MCNP/Serpent plugins don't provide much beyond the general features, there isn't much to describe here. But if we do expand the capabilities (e.g., more methods on the `Results` classes) in the future, then a description should be added here.